From 9ce0df1a9daf9a0320cdd83994ffef61278696ea Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Thu, 28 Jul 2016 22:33:07 -0400
Subject: [PATCH 1/3] set keyboard xlayouts with 'us' by default

Previously only graphic set keyboard xlayouts with 'us' by default,
and text mode did not have the setting and it caused 100% cpu when
loadkey invoked. The hung of loadkey also caused reboot failed.

The setting in default ks works on both of graphic and text.

Upstream-Status: Inappropriate [wrlinux specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 data/interactive-defaults.ks | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/interactive-defaults.ks b/data/interactive-defaults.ks
index f692ccd..cda7f76 100644
--- a/data/interactive-defaults.ks
+++ b/data/interactive-defaults.ks
@@ -2,3 +2,4 @@
 # This is not loaded if a kickstart file is provided on the command line.
 auth --enableshadow --passalgo=sha512
 firstboot --enable
+keyboard --xlayouts='us'
-- 
2.8.1

